core/modules/SConscript

```diff
@@ -66,19 +64,12 @@ shProducts = { "xrdsvc" : {'mods' : """global mysql proto
                            'distDir' : 'lib'
                            },
 
-               "_czarLib" : {'mods' : """ccontrol czar global
-                                         log
-                                         parser proto
-                                         qana qdisp qproc query
-                                         rproc sg""".split(),
+               "czarProxy" : {'mods' : """ccontrol czar global parser proto proxy qana
+                                          query qdisp qproc rproc sg""".split(),
                              'shmods' : ['css'],
-                             'libs' : """ssl crypto antlr log
-                                      mysqlclient_r protobuf
-                                      XrdClient XrdPosix XrdSsi XrdUtils
-                                      boost_regex boost_thread boost_system
-                                      boost_signals""".split(),
+                             'libs' : """antlr log protobuf XrdSsi boost_regex""".split(),
```

Comment:
I thought this libraries where mandatory for the build, does new gcc version change that?